kano_wifi_gui/PasswordScreen.py

```diff
@@ -141,7 +143,11 @@ def _on_connect(self, widget=None):
         )
 
     def _set_button_sensitive(self, widget, event):
-        self._connect_btn.set_sensitive(True)
+        '''
+        Enable the Connect button only if the passphrase is long enough
+        '''
+        passphrase_length=widget.get_text_length()
+        self._connect_btn.set_sensitive(passphrase_length >= self.passphrase_minlen)
```

Comment:
What happens when they are trying to connect to an Open network? 
Are we certain that all points have this min passphrase length of 8? WEP, WPA, WPA2?

Reply:
Also, should we show a message to the user about the min length such that it's clear why the button is disabled?

Reply:
You are right! WEP is either 5 or 13 in length. Discarding this PR and moving the logic to a lower layer.